### PR TITLE
split up the main file

### LINKED ISSUE #110: Split up main file
`grocy.py` is getting quite large and should be split up.

### PR DESCRIPTION
**Pull request by yellalena**
## Description

Split up the main file to multiple files.

fixes #110 

## Checklist
  - [x] The code change is tested and works locally.
  - [x] tests pass. **Your PR won't be merged unless tests pass**
  - [x] There is no commented out code in this PR

### CODE REVIEW

**Comment by yellalena** [comment]
@SebRut hey sir check this out :)

File: requirements-dev.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ coveralls~=2.1.2
 doctr~=1.8.0
 pdoc3~=0.9.1
 responses~=0.12.0
+tzlocal~=2.1
```
Comment (SebRut) [inline requirements-dev.txt:5]: Is this actually needed?

**Review by SebRut — APPROVED** [review]

File: requirements-dev.txt (hunk 1 above)
Comment (yellalena) [reply requirements-dev.txt:5]: @SebRut you are using tzlocal in utils.localize_datetime, which is then used in grocy_api_client.

**Comment by yellalena** [comment]
@SebRut could you please label the PR as 'hacktoberfest-accepted'? because I'm participating in the event 🥺 👉👈

**Pull request merged**